shell/key-bindings.zsh

```diff
@@ -103,8 +103,10 @@ fzf-history-widget() {
   local ret=$?
   if [ -n "$selected" ]; then
     num=$selected[1]
-    if [ -n "$num" ]; then
+    if [[ "$num" =~ ^-?[0-9]+$ ]]; then
```

Comment:
Do we need the `-?` part? In my case, `fc -rl 1` doesn't show any entries whose number starts with `-`.

Reply:
Yup, it's redundant, removed.

Reply:
Yup, it's redundant, removed.